core/runner.py

```diff
@@ -166,12 +176,23 @@ def _validate(self, job, custom_iterator=None):
         NOTE: Assumes 'weights' are the actual weights and not a path.
         """
         logging.info("Validating model...")
+        assert job.session_filepath, \
+            "Transformed training and test sets not created yet!"
+        train_dataset_path = os.path.join(
+                                job.session_filepath, 
+                                'train.csv'
+                            )
+        test_dataset_path = os.path.join(
+                                job.session_filepath, 
+                                'test.csv'
+                            )
+        data_count = count_datapoints(job.session_filepath)
```

Comment:
since it's here twice can we make this a helper function? def _helper_dataset() returns train_dataset_path, test_dataset_path, data_count. For lines 181-189 in this function, and these also appear in _train.

Reply:
Done.